src/Tizen.NUI.Components/Controls/NumberPicker.cs

```diff
@@ -0,0 +1,531 @@
+﻿/* Copyright (c) 2021 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+using System;
+using Tizen.NUI;
+using Tizen.NUI.BaseComponents;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+
+namespace Tizen.NUI.Components
+{
+    /// <summary>
+    /// NumberPickerValueChangedEventArgs is a class to notify changed NumberPicker value argument which will sent to user.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NumberPickerValueChangedEventArgs : EventArgs
+    {
+        /// <summary>
+        /// NumberPickerValueChangedEventArgs default constructor.
+        /// <param name="value">value of NumberPicker.</param>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]   
+        public NumberPickerValueChangedEventArgs(int value)
+        {
+            Value = value;
+        }
+
+        /// <summary>
+        /// NumberPickerValueChangedEventArgs default constructor.
+        /// <returns>The current value of NumberPicker.</returns>
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]   
+        public int Value { get; }
+        
+    }
+
+    /// <summary>
+    /// NumberPicker is a class which provides a function that allows the user to select 
+    /// a value through a scrolling motion by expressing the specified value as a list.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class NumberPicker : Control
```

Comment:
Could you override the method `void ApplyStyle(ViewStyle)`?

The method is called,
* when the theme applied in the constructor
* when the theme is changed when the user changed it through the settings app.
* it also ban be called by user explicitly who wants to change the look of existing component.

For example.
```C#
var numberPicker = new NumberPicker() { ...  };
//... 
numberPicker.ApplyStyle(new NumberPickerStyle() { /*...*/ });
```

Reply:
Done. Thank you for the guide.